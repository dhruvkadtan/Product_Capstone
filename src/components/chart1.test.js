import React from 'react';
import renderer from 'react-test-renderer';
import  Chart1 from './chart2';


  test('render correctly', () => {
    const tree = renderer.create(<Chart1/>)
    expect(tree).toMatchSnapshot();
  })


