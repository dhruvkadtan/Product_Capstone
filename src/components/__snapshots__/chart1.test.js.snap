// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
Array [
  <hr
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  />,
  <div
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <canvas
      height={150}
      role="img"
      width={300}
    />
  </div>,
  <h6
    className="react-reveal text-center mt-1 Playfair"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    Top Viewed Product
  </h6>,
  <hr
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  />,
]
`;
